packages/web3-eth-contract/src/proxies/MethodsProxy.js

```diff
@@ -191,7 +191,7 @@ export default class MethodsProxy {
         method.setArguments(methodArguments);
 
         // If no parameters are given for the eth_call or eth_send* methods then it will set a empty options object.
-        if (typeof method.parameters[0] === 'undefined') {
+        if (typeof method.parameters[0] === 'undefined' || method.parameters[0] === null) {
```

Comment:
Could you extend the test with the new case? :)

Reply:
@nivida Done!

Reply:
Great! Thanks for your changes:) 

Sorry if I annoy you but a test on ``falsy`` is probably the better solution. This will allow us to use ``false``, ``null`` and ``undefined`` without checking of the type.

``` javascript
if (method.parameters[0]) {
    method.parameters[0] = {};
}
```

Reply:
Don't worry :P This change is better than the initial ;)
Done!